scripts/ExternalInterfaces/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Fetch any externally-managed interfaces
+include ( ExternalProject )
+
+# mslice
+set ( _mslice_external_root ${CMAKE_CURRENT_BINARY_DIR}/mslice )
+ExternalProject_Add ( mslice
+  PREFIX ${_mslice_external_root}
+  GIT_REPOSITORY "https://github.com/mantidproject/mslice.git"
+  GIT_TAG master
+  GIT_SHALLOW 1
```

Comment:
For some reason the `git clone` fails for me unless I remove this (`GIT_SHALLOW`) and the next (`GIT_PROGRESS`) line. Otherwise the command in `mslice-gitclone.cmake` is:
```
COMMAND "/usr/bin/git" clone --origin "GIT_SHALLOW" "https://github.com/mantidproject/mslice.git" "GIT_PROGRESS"
```

instead of

```
COMMAND "/usr/bin/git" clone --origin "origin" "https://github.com/mantidproject/mslice.git" "mslice"
```